en/simulation/index.md

```diff
@@ -221,13 +218,13 @@ In this case the simulator and PX4 use the host system time and do not wait on e
 
 To disable lockstep in PX4, run `make px4_sitl_default boardconfig` and set the `BOARD_NOLOCKSTEP` "Force disable lockstep" symbol which is located under toolchain.
 
-To disable lockstep in Gazebo, edit [the model SDF file](https://github.com/PX4/PX4-SITL_gazebo-classic/blob/3062d287c322fabf1b41b8e33518eb449d4ac6ed/models/plane/plane.sdf#L449) and set `<enable_lockstep>false</enable_lockstep>`.
+To disable lockstep in Gazebo Classic, edit [the model SDF file](https://github.com/PX4/PX4-SITL_gazebo-classic/blob/3062d287c322fabf1b41b8e33518eb449d4ac6ed/models/plane/plane.sdf#L449) and set `<enable_lockstep>false</enable_lockstep>`.
```

Comment:
How do I disable lockstep in Gazebo (the models don't appear to contain a lockstep tag).

Reply:
@hamishwillee I just ran the sim based on @dakejahl's recent restructure of GZ bridge and service calls. I haven't had any issues with running a high RTF (no issues arming or flying).

I am not 100% sure if it's still needed, but from a user perspective I wouldn't know if anything is wrong without it

Reply:
With new gz you can't disable lockstep. Lockstep just means the PX4 time is synchronized with sim time. The GZBridge now sets the PX4 time on every sim step, in the gz clockCallback

Reply:
@dakejahl Thanks. I've split this out then such that Gazebo has its own section where it says:

>Gazebo runs the simulator at the same speed as PX4 (the GZBridge now sets the PX4 time on every sim step, in the `clockCallback`).
> 
> Lockstep cannot be disabled on Gazebo.

Then the existing material on disabling lockstep etc is all kept together under a Gazebo Classic/jMAVSim section. My assumption is that this has not changed for those simulators (right?). Note, I will be pushing that stuff out of this doc into individual simulators, but I want to confirm this first.

Reply:
That's correct, the other simulators have not changed